src/map/status.hpp

```diff
@@ -3162,7 +3163,14 @@ struct status_change {
 #ifndef RENEWAL
 	unsigned char sg_counter; //Storm gust counter (previous hits from storm gust)
 #endif
+private:
 	struct status_change_entry *data[SC_MAX];
+public:
+	status_change_entry * getSCE(enum sc_type type) const;
+	status_change_entry * getSCE(uint32 type) const;
+	status_change_entry * createSCE(enum sc_type type);
+	void deleteSCE(enum sc_type type);
+	void clearSCE(enum sc_type type);
```

Comment:
I think we should not differentiate between those two, as the future container should handle this later, dont you think?